sound/soc/sof/ipc4-topology.c

```diff
@@ -1318,6 +1378,41 @@ static int sof_ipc4_prepare_mixer_module(struct snd_sof_widget *swidget,
 	return sof_ipc4_widget_assign_instance_id(sdev, swidget);
 }
 
+static int sof_ipc4_prepare_src_module(struct snd_sof_widget *swidget,
+				       struct snd_pcm_hw_params *fe_params,
+				       struct snd_sof_platform_stream_params *platform_params,
+				       struct snd_pcm_hw_params *pipeline_params, int dir)
+{
+	struct snd_soc_component *scomp = swidget->scomp;
+	struct snd_sof_dev *sdev = snd_soc_component_get_drvdata(scomp);
+	struct sof_ipc4_src *src = swidget->private;
+	struct snd_interval *rate;
+	int ret;
+
+	src->available_fmt.ref_audio_fmt = &src->available_fmt.base_config->audio_fmt;
+
+	/*
+	 * only input format and sink rate are required to be sent to the FW for SRC,
+	 * so no need to initialize output format like copier.
+	 */
```

Comment:
similar suggestion, use the similar comment as used in pga and mixer:
`/* output format is not required to be sent to the FW for SRC */` ?

Reply:
sure